x-pack/plugins/security_solution/public/dashboards/pages/utils.ts

```diff
@@ -6,21 +6,13 @@
  */
 
 import type { ChromeBreadcrumb } from '@kbn/core/public';
-import { isEmpty } from 'lodash/fp';
 import type { RouteSpyState } from '../../common/utils/route/types';
 
 export const getTrailingBreadcrumbs = (params: RouteSpyState): ChromeBreadcrumb[] => {
-  let breadcrumb: ChromeBreadcrumb[] = [];
-
-  const dashboardTitle = params?.state?.dashboardTitle?.trim();
-  if (params?.state?.dashboardTitle || params.detailName) {
-    breadcrumb = [
-      ...breadcrumb,
-      {
-        text: !isEmpty(dashboardTitle) ? dashboardTitle : params.detailName,
-      },
-    ];
+  const breadcrumbName = params?.state?.dashboardName;
+  if (breadcrumbName) {
```

Comment:
Using `params.detailName` was causing a change (id -> name) in the last breadcrumb when the dashboard is finally loaded. 
I checked other places, the convention seems to be to show the last "name" breadcrumb only when the information is loaded.